app/locales/ro/import.json

```diff
@@ -0,0 +1,5 @@
+{
+    "import_title":"Importa Locatii (optional)",
+    "import_step_1":"1. Daca aveti locatii anterioare salvate in contul Google, le puteti importa logandu-va in contul Google si downloadand Istoricul Locațiilor",
```

Comment:
logandu-va in contul Google => prin autentificare la contul Google
downloadand => descarcand